Chinese/mayDay/mayDay.md

```diff
@@ -15,4 +15,4 @@
 就缺一个优秀的主唱了，要不要一起？”悟空不管不顾，继续自我陶醉地唱着。人群中窜出四个人，
 包括之前长发小眼和带眼镜的那两个男人，不约而同地说道：“就这么定了吧！对了，还要取个
 酷炫一点的艺名。要不，就叫阿信吧？”悟空突然停止了歌唱，回到：“不错！”
-悟空诺有所思的唱起了孙悟空
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.